features/orabos/file.include/etc/dracut.conf.d/99-disable-watchdog.conf

```diff
@@ -0,0 +1 @@
+omit_drivers+=" iTCO_wdt "
```

Comment:
what is the issue with the watchdog on your side? does it bevae wrong?

Reply:
watchdog seems to cause issues and did sometimes reset during boot